Using LinkedIn URL lib to detect links properly

joshi1999 opened the pull request.

joshi1999 committed:
Using LinkedIn URL lib to detect links properly

FireInstall commented:
![](https://github.trello.services/images/mini-trello-icon.png) [VentureChat: Links per LinkedIn Lib ordentlich erkennen](https://trello.com/c/p5SsjgbW/238-venturechat-links-per-linkedin-lib-ordentlich-erkennen)

Aust1n46 committed:
Support 1.20.4

Aust1n46 committed:
Bugfix use 1.20.4 json boolean type for urls

FireInstall committed:
use correct url-detector lib
update ProtocolLib, so it's compiling again.

FireInstall committed:
Made UrlDetector work with color codes.

use correct url-detector lib
update ProtocolLib, so it's compiling again.

FireInstall commented:
Do we want to make a PR upstream, so maybe we don't need to have to keep a version parallel ourselves?

joshi1999 commented:
https://github.com/Aust1n46/VentureChat/pull/93

Already happened xD

FireInstall committed:
add .iml to gitignore.
I don't know why intellij generates it in th first place, but all the methods of the bug tracker are outdated.

FireInstall committed:
Added way more comments to clarify what the code does
Fix wrongly overwriting every formatting code with the last one, even if they should work together like bold and underlined.
Note: As per Minecraft's custom colour codes overwrite all formatting!

FireInstall committed:
Merge pull request #2 from GreenSurvivors/colorCodes

Made UrlDetector work with color codes.
